www/_posts/2017-09-22-plugins-release.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Plugins Release"
+categories: news
+tags: release plugins
+---
+
+The following plugins were updated today because they are being deprecated:
+
+* cordova-plugin-console: 1.1.0
+* cordova-plugin-compat: 1.2.0
+* cordova-plugin-device-motion: 2.0.0
+* cordova-plugin-device-orientation: 2.0.0 
+
+`cordova-plugin-console` has been integrated into `cordova-ios@4.5.0+`. It is not needed anymore. Make sure to remove it from your projects if you plan on updating your `cordova-ios`!
+
+Similarly, `cordova-plugin-compat` has been integrated into the upcoming `cordova-android@6.3.0` release. Please remove it from your projects when you update to the latest version of `cordova-android`. 
```

Comment:
@stevengill this is a nitpick but you shouldn't need to remove the plugin from your project. It should just be skipped when it encountered.

Reply:
So if you already have `compat` installed in your project, and then you update to `cordova-android@6.3.0+`, it will cause build errors. Removing the plugin solves these problems. I believe this is because it is during the fetch phase of `plugin add` that does `cordovaDependency` and `engine` checks. Where adding a new android to an existing project is considered a plugin restore instead of a plugin add, so fetching is skipped

Reply:
oh upgrading, I never upgrade. I remove and add the new platform version. Carry on